server/coordinator/coordinator_headlessauth.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/suborbital/vektor/vk"
 )
 
+// nolint
```

Comment:
what's failing for lint? 🤔

Reply:
It's not currently being used; @rnpridgeon I think this is where you're going to be doing your work for MT authn/authz